src/test/java/com/google/api/generator/engine/ast/ClassDefinitionTest.java

```diff
@@ -347,4 +348,8 @@ private static ForStatement createForStatement() {
         .setBody(Arrays.asList(ExprStatement.withExpr(createAssignmentExpr())))
         .build();
   }
+  // Create a simple block comment to stand for the Apache License header.
+  private static List<CommentStatement> createFileHeader() {
```

Comment:
If this is just used once, do we need a helper method for it? Would prefer to just add the logic to the method where it's used.

Reply:
Updated with a unit test (nestClassWithFileHeader) which also uses this. So I guess we can keep it. But this similar helper is only called once in JavaWriterVisitorTest, so removed it there. Thanks